### PR TITLE
Automation Indicator

### PR DESCRIPTION
**Pull request by qwertyuiop888**
fixes: #7672 

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by qwertyuiop888** [comment]
Since we are just adding automation to the docs, it would be nice to have an easy way to view automation status on a website. This adds a small indicator next to the time, with a green check for automated tested and a grey check for tested by hand. 

It would be useful for users to see automation status, as they may want to contribute by writing a test after reading the page

Screen Shot 2020-07-14 at 2 56 01 PM

Screen Shot 2020-07-14 at 2 55 53 PM

File: layouts/partials/primary_top.html [hunk 1]
```diff
@@ -104,6 +104,26 @@ <h1 id="title">
                                         <span>&nbsp;</span>
                                         {{- printf (i18n "page_reading_time") .Page.ReadingTime -}}
                                     </span>
+
+                                    <span>&nbsp;</span>
+
+                                    <span>
+                                        {{if isset .Params "test"}}
+                                            {{ if(.Params.test) }}
```
Comment (frankbu) [inline layouts/partials/primary_top.html:112]: There a 3 possible values of .Params.test:
1. "yes"
2. "no"
3. "n/a"

1 should display green check, 2 should display grey check, 3 should display nothing (same as not isset).

Comment (qwertyuiop888) [reply layouts/partials/primary_top.html:112]: Ah sorry, The isset check around the code makes sure it is not displayed if "n/a"

**Comment by qwertyuiop888** [comment]
Im also pretty confused as to why reading the value yes / no as a boolean works. There does not seem to be any documentation or issues about it online.

File: i18n/en.toml [hunk 2]
```diff
@@ -297,3 +297,13 @@ other = "Document"
 
 [log_change]
 other = "Change"
+
+[tested]
+other = "tested"
+
+[has_test]
+other = "An automated test is available for this document."
```
Comment (frankbu) [inline i18n/en.toml:305]:
```suggestion
other = "An automated test is available for this page. Click for details."
```

File: i18n/en.toml [hunk 3]
```diff
@@ -297,3 +297,13 @@ other = "Document"
...
+
+[needs_test]
+other = "This document has only been tested by hand. Help create an automated test!"
```
Comment (frankbu) [inline i18n/en.toml:308]:
```suggestion
other = "No automated test is available for this page. Click for details or to help create one."
```

**Comment by frankbu** [comment]
@qwertyuiop888 From the feedback we've heard so far, I think we need to be careful to not imply anything about how well a page is tested (either manual or auto). We should make it clear that this is just an indication of whether or not there is an automated test for the page or not. So, in that light, my recommendation is to change the tag from `TESTED` to just `TEST` or maybe (`PAGE TEST`). This is more clearly saying whether or not there is a test case for the doc, not whether or not the doc has been tested one way or another.

Using that tag, then I think we should use an x icon for the "no" case (instead of checkmark), but make the x grey, instead of red (because it's still true that red x generally implies something is broken).

- test: yes ->  (green check) PAGE TEST
- test: no -> (grey x) PAGE TEST

How about changing it like this (and the hover text as I suggested in in-line comments) and then we can merge it and see how it goes. If we get more feedback or better ideas later, we can change it in a future PR. WDYT?

**Comment by qwertyuiop888** [comment]
That sounds good. It also solves the visibility issues with the same icons!

**Pull request merged**